index.html

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+  <body>
+    <header class="toolbar-container">
+      <a href=""
+        ><img id="logo" src="./images/header/My Logo.png" alt="logo"
+      /></a>
+      <nav>
+        <a href="">
+          <img
+            id="nav-logo"
+            src="./images/header/Hamburger_icon.svg.png"
+            alt="logo" />
+        </a>
+      </nav>
+    </header>
+    <main id="headline">
+      <section>
+        <div id="name"><h1>I’m Ali Hassan. Glad to see you!</h1></div>
+        <div id="introduction">
+          <p>
+            I'm a software developer! I can help you build a product, feature or
+            website Look through some of my work and experience! If you like
+            what you see and have a project you need coded, don't hesitate to
+            contact me.
+          </p>
+        </div>
+        <div id="connect">
+          <img src="./images/main/LET’S CONNECT.svg" alt="" />
+        </div>
```

Comment:
It would be better to use a `p` tag with text instead of `img` tag because it's not an image🤔🤔

Reply:
I did the required changes.